kedro/framework/cli/registry.py

```diff
@@ -48,3 +50,33 @@ def describe_registered_pipeline(
     result = {"Nodes": nodes}
 
     click.echo(yaml.dump(result))
+
+
+@command_with_verbosity(registry, "export")
+@click.argument("name", nargs=1, default="__default__")
+@click.option(
+    "-o",
+    "--output-file",
+    help="File path to export the pipeline to. Suffix must be .json",
+    type=click.Path(dir_okay=False, writable=True, path_type=Path, resolve_path=True),
+)
+@click.pass_obj
+def export_registered_pipeline(
+    metadata: ProjectMetadata, name: str, output_file: Path, **kwargs
+) -> None:  # noqa: unused-argument, protected-access
+    """Export a registered pipeline by providing a pipeline name."""
+    if output_file.suffix != ".json":
+        raise KedroCliError(
+            f"Output file must be a .json file. Received: '{output_file}'"
+        )
+
+    pipeline_obj = pipelines.get(name)
```

Comment:
What are your thoughts on making this `name=None` and defaulting it to the `__default__` option if not provided?

Reply:
Makes sense, will do 👍

Reply:
I guess the point is that the CLI command doesn't require a pipeline name

Reply:
Yes, currently, it does not require a name (defaults to `__default__`). This is the behaviour we want, right?

Reply:
Nice!